x-pack/legacy/plugins/integrations_manager/server/plugin.ts

```diff
@@ -19,22 +22,39 @@ export interface CoreSetup {
   http: HttpServiceSetup;
 }
 
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface PluginInitializerContext {}
```

Comment:
Keep the `export` of `PluginInitializerContext`.

Would now be 

`export { PluginInitializerContext } from 'src/core/server'`